transport/tchannel/outbound.go

```diff
@@ -64,6 +64,17 @@ type outbound struct {
 	HostPort string
 }
 
+func (o outbound) Start() error {
+	// TODO: Should we create the connection to HostPort (if specified) here or
```

Comment:
It's probably best to start lazy - wait for the first call.

@prashantv @akshayjshah might be able to shed some light here.